deployments/bsctestnet_addresses.json

```diff
@@ -99,7 +99,7 @@
     "RewardFacet": "0x905006DCD5DbAa9B67359bcB341a0C49AfC8d0A6",
     "SXP": "0x75107940Cf1121232C0559c747A986DEfbc69DA9",
     "SetCheckpointBsctestnet": "0x39B532d141f3243DF455a3Fb30307A9d556bBaDe",
-    "SetterFacet": "0x490DFD07f592452307817C4283866035BDb3b275",
+    "SetterFacet": "0xb619F7ce96c0a6E3F0b44e993f663522F79f294A",
```

Comment:
Verify all newly deployed facets (MarketFacet , PolicyFacet, SetterFacet)

Reply:
It's done